src/lib/reminderFields.ts

```diff
@@ -26,6 +26,23 @@ export const buildReminderFields = (today: Date = new Date()): ReminderFields =>
     };
 };
 
-export const getReminderFields = (reminderFields?: ReminderFields): ReminderFields => {
+const US_EOY_APPEAL_REMINDER_FIELDS: ReminderFields = {
+    reminderCTA: 'Remind me late December',
+    reminderDate: `2020-12-28 00:00:00`,
+    reminderDateAsString: `in late December`,
```

Comment:
should we also drop the 'in' here?

Reply:
@TomPretty @ionamckendrick there's actually some logic that inserts "in" if you leave it out here!
Anyway, since we have the space here, isn't it preferable to use better language?!